_includes/footer.html

```diff
@@ -13,6 +13,14 @@
             {% endfor %}
           </div>
         </div>
+        <div class="logos">
```

Comment:
Hey @s-kennedy. For Pittsburgh we should just use the footer without the EU info